server.py

```diff
@@ -119,6 +126,20 @@ def get_metrics():
     level= logging.ERROR - (args.verbose*10),
 )
 
+def consumer():
+    while True:
+        alias = alias_queue.get()
+        if alias is None:  
+            break  
+        try:
+            increment_used_column(DATABASE_FILE, alias)
```

Comment:
inside of modules/metrics.py can we add a new `prometheus_client.Gauge` metric called `used_alias_queue_size`? then right above the call to `increment_used_column`

Reply:
so basically
```py
MetricsHandler.used_alias_queue_size.set(len(alias_queue)
try:
    with MetricsHandler.query_time.labels("increment_used").time():
        increment_used_column(DATABASE_FILE, alias)
except Exception:
    # rest of code
```